### PR TITLE
EGO Resprite, Adds Syrinx.

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds some outstanding EGO resprites, and adds in syrinx

## Changelog
:cl:
add: Added syrinx gun
tweak: resprited old EGO sprites
/:cl:

### CODE REVIEW

**Review by dsmith328 — APPROVED** [review]
Keep me up to date on funny SFX for Syrinx

File: icons/mob/clothing/ego_gear/suit.dmi
Comment (dsmith328) [inline icons/mob/clothing/ego_gear/suit.dmi:1]: Bayard Office Suit unnamed

File: icons/obj/clothing/ego_gear/suits.dmi
Comment (dsmith328) [inline icons/obj/clothing/ego_gear/suits.dmi:1]: Same

File: icons/obj/ego_weapons.dmi
Comment (dsmith328) [inline icons/obj/ego_weapons.dmi:1]: Mimicry Scythe but not in hand?

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: icons/obj/ego_weapons.dmi
Comment (Kitsunemitsu) [reply icons/obj/ego_weapons.dmi:1]: shiiiiit I will change that

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**